README.md

```diff
@@ -100,7 +115,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is the representation of a plan or theory in the form of a outline or model.
+We represent 1-many relationships in a example like Orange Tree vs. oranges. An orange tree can grow many oranges. But oranges can only grow on a orange tree.  
+
 ```
```

Comment:
Great example! In the context of web development, a schema has a more specific definition. It maps your ERD onto the database by describing the entities' attributes and relationships as tables, in the language of the database. In our case, that language is SQL.